src/commands/postDeploy/checkLinuxWebAppDownDetector.ts

```diff
@@ -50,26 +41,34 @@ export async function checkLinuxWebAppDownDetector(correlationId: string, node:
             }
 
             if (Date.now() > detectorTimeoutMs) {
-                if (showOutput) {
-                    const noIssuesFound: string = `Diagnosing for "${node.root.client.siteName}" has timed out.`;
-                    ext.outputChannel.appendLog(noIssuesFound);
-                    context.telemetry.properties.timedOut = 'true';
-                }
+                context.telemetry.properties.timedOut = 'true';
                 return undefined;
             }
 
-            // tslint:disable-next-line: no-unsafe-any
-            detectorErrorMessage = await getLinuxDetectorError(context, linuxContainerStartFailureId, node, deployResultTime, node.root.client.fullName);
+            // time constraint of being less than the current time by 14.5 minutes is enforced system wide for all detectors
+            // however LinuxLogViewer detector ignores it and gets the most recent data
+            const nowTime: number = Date.now();
+            const startTimeDate: Date = new Date(nowTime - (60 * 60 * 1000));
+            const endTimeDate: Date = new Date(nowTime - (30 * 60 * 1000));
+
+            const timeFormat: string = 'YYYY-MM-DDTHH:mm';
+            const startTime: string = moment.utc(startTimeDate).format(timeFormat);
+            const endTime: string = moment.utc(endTimeDate).format(timeFormat);
+
+            detectorErrorMessage = await getLinuxDetectorError(context, linuxLogViewer, node, startTime, endTime, deployment.endTime);
 
             if (!detectorErrorMessage) {
-                // poll every minute
-                await delay(1000 * 60);
+                // poll every 10 seconds
+                await delay(1000 * 10);
             }
         }
 
+        const enableDetectorsSetting: string = 'enableDetectors';
+        const showOutput: boolean | undefined = getWorkspaceSetting<boolean>(enableDetectorsSetting);
         if (showOutput) {
-            await ext.ui.showWarningMessage(detectorErrorMessage, { title: 'Open in Portal' });
-            await openInPortal(node.root, `${node.root.client.id}/troubleshoot`, { queryPrefix: `websitesextension_ext=asd.featurePath%3Ddetectors%2F${linuxContainerStartFailureId}` });
+            ext.outputChannel.appendLog(detectorErrorMessage);
+            await ext.ui.showWarningMessage(detectorErrorMessage, { title: 'View details' });
```

Comment:
You'll want to do something like this where it doesn't await the message and creates a new event for the result of the message:
https://github.com/microsoft/vscode-azureappservice/blob/d6fbafdb8c8aac22c3310ec25f932fe62894db99/src/commands/deploy/showDeployCompletedMessage.ts#L22

Otherwise we won't get any telemetry for users when an error is found, but they never interact with the notification